gateway-ha/src/main/java/io/trino/gateway/ha/router/TrinoQueryProperties.java

```diff
@@ -114,7 +116,8 @@ public TrinoQueryProperties(
             @JsonProperty("schemas") Set<String> schemas,
             @JsonProperty("catalogSchemas") Set<String> catalogSchemas,
             @JsonProperty("isNewQuerySubmission") boolean isNewQuerySubmission,
-            @JsonProperty("isQueryParsingSuccessful") boolean isQueryParsingSuccessful)
+            @JsonProperty("isQueryParsingSuccessful") boolean isQueryParsingSuccessful,
```

Comment:
Why do we still need this field? I thought just having the getter is sufficient.

Reply:
@ebyhr PTAL if I understood correctly, when you mean just having getter.